app/utils/estimate_prioritization/estimate_prioritization.py

```diff
@@ -0,0 +1,113 @@
+import pandas as pd
+from statsmodels.stats.proportion import proportion_confint
+from app.utils.estimate_prioritization import prioritization_criteria_testunadj, \
+    prioritization_criteria_testadj, pooling_function_maps
+
+
+def get_pooled_estimate(estimates):
+
+    if estimates.shape[0] == 1:
+        pooled = estimates.iloc[0]
+    else:
+        # use the estimate with the max denominator as the 'base' for the pooled estimate
+        pooled = estimates.loc[estimates['denominator_value'].idxmax(axis = 1)] 
+
+        # pool all values that can be pooled
```

Comment:
can we add better comments please from line 15 - 40? lacking comments and comments that exist are weak :/

Reply:
@rahularoradfs care to take a crack

Reply:
taking a crack now, give me 10

Reply:
"pool all values that can be pooled"

Reply:
done